source/form.html

```diff
@@ -0,0 +1,182 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Кэт энерджи</title>
+</head>
+
+<body class="page__body">
+  <header class="main-header">
+    <a class="main-header__logo" href="index.html">
+      <img class="main-header__logo-image" src="img/logos/logo.svg" width="33" height="38"
+        alt="Логотип Cat Energy картинка рыжего кота в плаще.">
+    </a>
+    <a class="main-header__logo" href="index.html">
+      <img class="main-header__logo-text" src="img/logos/text-logo.svg" width="101" height="18"
+        alt="Логотип Cat Energy текстовая надпись.">
+    </a>
+    <nav class="main-nav">
+      <button class="main-nav__toggle" type="button">
+        <span class="visually-hidden">Открыть меню.</span>
+      </button>
+      <ul class="site-list">
+        <li class="site-list__item">
+          <a href="index.html">
+            Главная
+          </a>
+        </li>
+        <li class="site-list__item site-list__item--active">
+          <a>
+            Каталог продукции
+          </a>
+        </li>
+        <li class="site-list__item">
+          <a href="form.html">
+            Подбор программы
+          </a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <main class="main-container">
+    <section class="form-headline">
+      <h1 class="form-headline__title">Подбор программы</h1>
+      <p class="form-headline__offer">Заполните анкету, и мы подберем<br>программу питания для
+        вашего кота</p>
+      <form class="form" action="https://echo.htmlacademy.ru/" method="get">
```

Comment:
```suggestion
      <form class="form" action="https://echo.htmlacademy.ru/" method="post">
```

много данных, лучше постом отправлять